### DISCUSSION TITLE
2 different BMS types?

### DISCUSSION THREAD

**Discussion by lennycb** [Louisvdw/dbus-serialbattery#329] [Q&A] [answered]
Hello, I have 2 questions that only partially concern your driver:

I have a battery pack 16x EVE280Ah. Since the cell capacity differed, I set the total capacity to the lowest value (271Ah). BMS is a DALY 400A.

Now I want to build a 2nd pack with 16x CATL (280Ah) or 16x EVE cells (280Ah) but probably with a JBD-BMS (are probably better than DALY)

Now my questions:
1. Can your driver handle the two BMS at the same time?
2. Should I also set the 2nd battery to 271Ah (I had read that the capacity should always be the same when connected in parallel.)

**Comment by Louisvdw** [comment, marked as answer]
You can have 2 BMS/batteries and you will see both in the Remote Console and in VRM cloud. But they are not yet combined to give you one large battery in the battery display/SOC. You will have to pick one for that.
Subscribe to #8 for when the multi bank feature will land.

To make things simpler it is always suggested to have 2 of the same batteries sets (cells + BMS), but that does not mean it will not work if they are not the same. You just might not get the full capacity of both. 

Different BMS will have some differences in how they calculate SOC, or trigger alarms, etc. So having the same BMS in both makes that easier. Also some BMS have settings that others do not have, making setup more limiting.

But having some variations is not critical. Having some differences is cells is nartural. They don't all age the same. 
Use the actual value (280Ah) for the new set. It is only for reporting the available capacity and will not change the actual capacity in the battery.

PS. If I hade to choose I would get 2x JBD BMS and not 2x Daly. I would also rather mix (JBD + Daly) than getting a extra Daly. If you want to keep the Daly, concider getting an external balancer.

**Comment by lennycb** [comment]
![image](https://user-images.githubusercontent.com/23618370/206681535-08a32c7f-84b7-4f6f-bca8-73b6d2005c8c.png)
Yes, I installed an external, active balancer.

In the meantime I saw that DALY offers a 5th generation BMS. This then has a "parallel module" which limits the currents between the individual banks to 15A.

You can probably also see in the app that there is a "balancing" between the benches.
Sounds exciting too. Does the good work? hmm...
Is probably something if you want to switch banks with large capacity differences together.

https://www.dalyelec.cn/newsshow.php?cid=24&id=85&lang=1

**Comment by lennycb** [comment]
@Louisvdw  have now searched a bit and came across SEPLOS. You can connect them to each other (via RS485) and via CAN to a Victron inverter.
The total capacity is displayed in the remote console.

Unfortunately, the remaining functions do not close. Maybe you know something there.

- What is the advantage of the BMS Master/Slvae system?
- Do the BMS regulate the equalizing currents between the battery banks? (I think it's good!)

I think that since that is no longer possible via your driver, I can no longer load under voltage control. (that would be a shame)

When do you think the topic "Multi-Bank-System" will be finished?
